### PR TITLE
Convert to esql style

### PR DESCRIPTION
**Pull request by mrallen1**

### CODE REVIEW

File: src/bh_route_vars.erl [hunk 1]
```diff
@@ -13,22 +13,18 @@
...
-
-    {ok, S2} = epgsql:parse(Conn, ?S_VAR,
-                            [?SELECT_VARS_BASE,
-                             "where v.name = $1"
-                            ], []),
-
-    #{?S_VAR_LIST => S1,
-      ?S_VAR => S2
-     }.
+    PrivDir = code:priv_dir(blockchain_http),
```
Comment (madninja) [inline src/bh_route_vars.erl:17]: Can we pull this into a db_worker helper function and use in at least two places ?

File: priv/vars.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+-- :vars_list
+SELECT v.name, v.type, v.value from vars_inventory v order by name
+
+-- :vars_get
+SELECT v.name, v.type, v.value from vars_inventory v where v.name = ?
```
Comment (madninja) [inline priv/vars.sql:5]: I believe this needs to be a $1 instead of `?`

**Review by madninja — CHANGES_REQUESTED** [review]

File: src/bh_route_vars.erl (hunk 1 above)
Comment (mrallen1) [reply src/bh_route_vars.erl:17]: yeah good idea

File: priv/vars.sql [hunk 3]
```diff
@@ -2,4 +2,4 @@
 SELECT v.name, v.type, v.value from vars_inventory v order by name
 
 -- :vars_get
-SELECT v.name, v.type, v.value from vars_inventory v where v.name = ?
+SELECT v.name, v.type, v.value from vars_inventory v where v.name = ?;
```
Comment (madninja) [inline priv/vars.sql:5]: No.. I meant replace ? with $1?

Comment (mrallen1) [reply priv/vars.sql:5]: I guess I wasn't sure where in esql file I should put a semicolon if anywhere? That's what I was trying to figure out

**Review by madninja — APPROVED** [review]

**Pull request merged**